### PR TITLE
refactor code

### PR DESCRIPTION
**Pull request by sirojan24**
Fix for the review comments

### CODE REVIEW

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/subscription/AndesSubscriptionManager.java [hunk 1]
```diff
@@ -395,13 +400,14 @@ private void removeLocalSubscriptionAndNotify(AndesSubscription subscription) th
...
-
+        // query for CQ engine to get the active subscriptions based on the filtering parameters.
+        // since there is a need for contains search in binding key, contains operation of CQ engine is used in query
         Query<AndesSubscription> subscriptionQuery = and
                 (equal(AndesSubscription.DURABILITY, isDurable), equal(AndesSubscription.PROTOCOL, protocolType),
                  equal(AndesSubscription.ROUTER_NAME, messageRouter),
                  contains(AndesSubscription.ROUTING_KEY, bindingKeyPattern.toLowerCase()));
 
-        if(!SELECT_ALL.equals(connectedNode)){
+        if(!SELECT_ALL_NODES.equals(connectedNode)){
```
Comment (cnapagoda) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/subscription/AndesSubscriptionManager.java]: Code formatting issue.

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/subscription/AndesSubscriptionManager.java [hunk 2]
```diff
@@ -430,16 +436,16 @@ private void removeLocalSubscriptionAndNotify(AndesSubscription subscription) th
         List<AndesSubscription> allInactiveSubscriptions = getInactiveSubscriberRepresentations();
 
         for (AndesSubscription inactiveSubscription : allInactiveSubscriptions) {
-            if(inactiveSubscription.getStorageQueue().getMessageRouterBindingKey().toLowerCase().
-                    contains(bindingKeyPattern.toLowerCase())){
+            if(StringUtils.containsIgnoreCase(inactiveSubscription.getStorageQueue().getMessageRouterBindingKey(),
```
Comment (cnapagoda) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/subscription/AndesSubscriptionManager.java]: Code formatting issue.

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/subscription/AndesSubscriptionManager.java [hunk 3]
```diff
@@ -467,8 +473,8 @@ private void removeLocalSubscriptionAndNotify(AndesSubscription subscription) th
             for (AndesSubscription inactiveSubscription : subscriptions) {
                 if(inactiveSubscription.getStorageQueue().getMessageRouter().getName().equals(messageRouter)
```
Comment (cnapagoda) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/subscription/AndesSubscriptionManager.java]: Code formatting issue.

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/subscription/AndesSubscriptionManager.java [hunk 4]
```diff
@@ -467,8 +473,8 @@ private void removeLocalSubscriptionAndNotify(AndesSubscription subscription) th
...
                         && inactiveSubscription.getProtocolType().equals(protocolType)
-                        && inactiveSubscription.getSubscriptionId().toLowerCase()
-                        .contains(subscriptionIdPattern.toLowerCase())){
+                        && StringUtils.containsIgnoreCase(inactiveSubscription.getSubscriptionId(),
+                        subscriptionIdPattern)){
```
Comment (cnapagoda) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/subscription/AndesSubscriptionManager.java]: Code formatting issue.

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/subscription/AndesSubscriptionManager.java [hunk 5]
```diff
@@ -338,13 +343,13 @@ private void removeLocalSubscriptionAndNotify(AndesSubscription subscription) th
...
         String messageRouter = destinationType.getAndesMessageRouter();
-
+        // query for CQ engine to get the active subscriptions based on the filtering parameters
         Query<AndesSubscription> subscriptionQuery = and
                 (equal(AndesSubscription.DURABILITY, isDurable), equal(AndesSubscription.PROTOCOL, protocolType),
                  equal(AndesSubscription.ROUTER_NAME, messageRouter),
                  equal(AndesSubscription.ROUTING_KEY, bindingKeyPattern.toLowerCase()));
 
-        if(!SELECT_ALL.equals(connectedNode)){
+        if(!SELECT_ALL_NODES.equals(connectedNode)){
```
Comment (cnapagoda) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/subscription/AndesSubscriptionManager.java]: Code formatting issue.

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesMessageMetadata.java [hunk 6]
```diff
@@ -483,8 +483,8 @@ public boolean equals(Object o) {
         if (!(o instanceof AndesMessageMetadata)){
```
Comment (cnapagoda) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesMessageMetadata.java]: Code formatting issue.

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/DeliveryResponsibility.java [hunk 7]
```diff
@@ -20,30 +20,28 @@
...
 
     /**
-     * Set the next delivery responsibility
+     * Set the next delivery responsibility.
      *
-     * @param deliveryResponsibility Responsibility that need to
-     *                               be performed in the delivery path
+     * @param deliveryResponsibility responsibility that need to be performed in the delivery path
      */
     public void setNextDeliveryFilter(DeliveryResponsibility deliveryResponsibility){
```
Comment (cnapagoda) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/DeliveryResponsibility.java]: Code formatting issue.

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/ExpiredMessageHandler.java [hunk 8]
```diff
@@ -35,33 +34,29 @@
     private PreDeliveryExpiryMessageDeletionTask preDeliveryExpiryMessageDeletionTask;
 
     /**
-     * Set the expiry message deletion task
-     * @param task Deletion task
+     * Set the expiry message deletion task.
+     *
+     * @param task deletion task
      */
     public void setExpiryMessageDeletionTask(PreDeliveryExpiryMessageDeletionTask task){
```
Comment (cnapagoda) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/ExpiredMessageHandler.java]: Code formatting issue.

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/subscription/AndesSubscriptionManager.java (hunk 1 above)
Comment (sirojan24) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/subscription/AndesSubscriptionManager.java]: Fixed in https://github.com/wso2/andes/pull/738

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/subscription/AndesSubscriptionManager.java (hunk 2 above)
Comment (sirojan24) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/subscription/AndesSubscriptionManager.java]: Fixed in https://github.com/wso2/andes/pull/738

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/subscription/AndesSubscriptionManager.java (hunk 3 above)
Comment (sirojan24) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/subscription/AndesSubscriptionManager.java]: fixed in https://github.com/wso2/andes/pull/738

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/subscription/AndesSubscriptionManager.java (hunk 4 above)
Comment (sirojan24) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/subscription/AndesSubscriptionManager.java]: fixed in https://github.com/wso2/andes/pull/738

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesMessageMetadata.java (hunk 6 above)
Comment (sirojan24) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesMessageMetadata.java]: fixed in https://github.com/wso2/andes/pull/738

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/ExpiredMessageHandler.java (hunk 8 above)
Comment (sirojan24) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/ExpiredMessageHandler.java]: fixed in https://github.com/wso2/andes/pull/738

**Pull request merged**